tmp/build/patch-grafana.sh

```diff
@@ -73,6 +73,14 @@ function grafana_patch_deployment() {
   ${HELM_DIR}/istio/charts/grafana/templates/deployment.yaml
 
   sed -i -e '/securityContext/,/fsGroup/d' ${HELM_DIR}/istio/charts/grafana/templates/deployment.yaml
+
+  # Fix for MAISTRA-746, can be removed when we move to Istio-1.2
+  grep -q 'maxUnavailable' ${HELM_DIR}/istio/charts/grafana/templates/deployment.yaml || \
```

Comment:
So, can we get rid of this check as discussed?

Reply:
if you really want to, sure